pkgs/desktops/gnustep/back/default.nix

```diff
@@ -9,18 +10,19 @@
 , libXmu
 }:
 
-gsmakeDerivation rec {
+mkDerivation (self: {
```

Comment:
I'd prefer these derivations to use `stdenv.mkDerivation`, since `pkgs/desktops/gnustep/mkDerivation.nix` looks over-engineered (see #180841 for a Qt equivalent).

Reply:
As in, it's unnecessary because at this point all it still does is add meta attrs?

Reply:
Yes.

Reply:
Works for me. I left it because iirc that's what KDE packages do too and it technically reduces code duplication. But yeah it's not really worth it for such a small package set.

Similarly, what do you think about the stdenv alias (in default.nix)?